.rubocop.yml

```diff
@@ -16,6 +16,8 @@ AllCops:
 
 Metrics/MethodLength:
   Max: 25
+  Exclude:
+    - 'app/workers/link_check_worker.rb'
```

Comment:
Some refactoring required? I don't really like adding exceptions to this `.rubocop.yml`, I think I'd even prefer a simple `#rubocop:disable` directly in the file.

Reply:
I can move them to `.rubocop_todo.yml` so that it can be considered for refactoring.

Reply:
Yeah but why not refactoring now that you are working on it? :/